tox.ini

```diff
@@ -7,7 +7,7 @@ env_list = coverage, build, firmware, docs, format, type, py{39,310,311,312}, vi
 # For instance, to install ingenialink from a custom branch.
 # It should be empty in production.
 [develop]
-    ingenialink = git+https://github.com/ingeniamc/ingenialink-python@develop
+    ingenialink = git+https://github.com/ingeniamc/ingenialink-python@INGK-892-add-boot-in-app-flag-in-the-load-firmware-of-ethercat
```

Comment:
Undo this change when https://github.com/ingeniamc/ingenialink-python/pull/414 is merged